src/style.css

```diff
@@ -0,0 +1,121 @@
+* {
+  padding: 0;
```

Comment:
```suggestion
```
no need to reset paddings for all selectors. make it just elements where it is necessary